framer/InputLayer.coffee

```diff
@@ -0,0 +1,151 @@
+Utils = require "./Utils"
+
+{Layer} = require "./Layer"
+{TextLayer} = require "./TextLayer"
+{Events} = require "./Events"
+
+Events.EnterKey  = "EnterKey"
+Events.BackSpaceKey  = "BackSpaceKey"
+Events.InputValueChange = "InputValueChange"
+Events.InputFocus = "InputFocus"
+Events.InputBlur = "InputBlur"
+
+class exports.InputLayer extends TextLayer
+
+	constructor: (options={}) ->
+
+		_.defaults options,
+			backgroundColor: "#FFF"
+			width: 500
+			height: 100
+
+		super options
+
+		# Layer containing input element
+		@input = new Layer
```

Comment:
Yes, technically true, but isn't it the clearest/shortest one available that's easy to target/select should you want to customize it? What alternatives would you suggest?